### ISSUE TITLE
Screen glitching on AMD Radeon

### ISSUE THREAD

**Issue by Paulporcupine** [Tyrrrz/LightBulb#252]
### Version

2.4.5

### Details

Screen randomly glitching mainly green and black 
Usually more apparent when a big change happens / new app open or close
example [https://cdn.discordapp.com/attachments/335127475589742593/1087223219494342736/PXL_20230320_035413333.TS2.mp4](url)
Amd 6650 xt 
Ryzen 7 5700x
Latest Drivers

### Steps to reproduce

Have lightbulb open

**Comment by Tyrrrz** [comment]
That's a very weird behavior. How is your monitor connected to the GPU?

**Comment by Potatocrip** [comment]
Can confirm, I've been having the same issue as well.  One monitor connected via HDMI and the other by displayport directly to my video card. 
The easiest way to reproduce this, from what I've seen, is by having two separate apps on either monitor with at least one of them utilizing the GPU to a decent extent. Then, use Alt+Tab to switch between those two apps. The artifacting will usually occur on the app that utilizes the GPU the most, if not (occasionally) both apps.
This artifacting only occurs when Lightbulb is running. 

AMD Radeon RX 6600 XT
Adrenalin Version 23.5.2 (5/23/2023)

**Comment by pbaykalov** [comment]
I recently learned that AMD has obscure colour management built into Radeon drivers. I can't be sure but it could be an issue.

https://pcmonitors.info/articles/taming-the-wide-gamut-using-srgb-emulation/

**Comment by Rockin132** [comment]
> I recently learned that AMD has obscure colour management built into Radeon drivers. I can't be sure but it could be an issue.
> 
> [pcmonitors.info/articles/taming-the-wide-gamut-using-srgb-emulation](https://pcmonitors.info/articles/taming-the-wide-gamut-using-srgb-emulation/)

Thank you, this has been driving me crazy for months trying to figure out the cause.

**Comment by Tyrrrz** [comment]
Were you able to resolve the issue somehow?

**Comment by Rockin132** [comment]
> Were you able to resolve the issue somehow?

I was not sadly, ended up uninstalling Lightbulb as I presumed it wouldn't be able to be fixed or even investigated until you were in a better safety situation and I realised a niche software glitch was small shakes in the face of that and put it to one side :)

**Comment by Tyrrrz** [comment]
Yeah, unfortunately hardware/driver-specific issues are difficult to investigate.

**Comment by pbaykalov** [comment]
I might say that I had problems with RX6950XT until I disabled the said colour management. If there is any other software which manipulates VCGT it will cause flickering too. For example, there's DisplayCal profile loader which uses it too. Windows display calibration also uses it.

**Comment by Tyrrrz** [comment]
Closing since there's no way for me to investigate this, and the original video (linked from Discord) has expired and is no longer available.
